app/backplane/radio_module/src/c_lora_transmit_tenant.cpp

```diff
@@ -63,28 +68,28 @@ void CLoraTransmitTenant::LandedRun() {
     NTypes::RadioBroadcastData data{};
 
     if (readTransmitQueue(data) && data.port == NNetworkDefs::RADIO_MODULE_GNSS_DATA_PORT) {
-        transmit(data);
+        (void) transmit(data);
```

Comment:
should we check the errors here and line 62? seems odd to add an error code here and then ignore it in most cases

Reply:
I originally did. Then I changed it last minute, because there's like no other code executing after this / depending on transmit to be successful, and we already log the errors in the transmit function. Also, I haven't really encountered a transmit error (yet, knock on wood). Worst case, a packet or two doesn't go through and we just keep going